cli/cmd/play.go

```diff
@@ -78,6 +79,45 @@ func internalPlayModule(cmdCtx *cmdcontext.CmdCtx, args []string) error {
 			"or directory")
 	}
 
+	// FillCtx returns error if no instances found.
+	var runningCtx running.RunningCtx
+
+	if err := running.FillCtx(cliOpts, cmdCtx, &runningCtx, []string{args[0]}, false); err == nil {
+		if len(runningCtx.Instances) > 1 {
+			return util.InternalError(
+				"Internal error: specify instance name",
+				version.GetVersion)
+		}
+
+		_, err := os.Stat(runningCtx.Instances[0].BinaryPort)
+		if err != nil {
+			return util.InternalError(
+				"Internal error: application binary port does not exist: %s",
+				version.GetVersion, err)
+		}
+
+		args[0] = runningCtx.Instances[0].BinaryPort
+
+	} else if libconnect.IsCredentialsURI(args[0]) {
+		if playUsername != "" || playPassword != "" {
+			return errors.New("username and password are specified with" +
+				" flags and a URI")
+		}
+		_, user, pass := libconnect.ParseCredentialsURI(args[0])
```

Comment:
Why do we not update a passed URI?

Reply:
Hm, [cause netbox module accepts URI with credentials](https://www.tarantool.io/en/doc/latest/platform/configuration/configuration_code/#listen-uri).

Reply:
So why do we set username/password here? It is a bit confusing. Let cut credentials from the URI.

Reply:
Ok.